packages/app-check/src/api.test.ts

```diff
@@ -118,6 +120,40 @@ describe('api', () => {
         })
       ).to.equal(appCheckInstance);
     });
+    it('starts debug mode on first call', async () => {
+      let token: string = '';
+      const fakeWrite = (tokenToWrite: string): Promise<void> => {
+        token = tokenToWrite;
+        return Promise.resolve();
+      };
+      stub(indexeddb, 'writeDebugTokenToIndexedDB').callsFake(fakeWrite);
+      stub(indexeddb, 'readDebugTokenFromIndexedDB').resolves(token);
+      const consoleStub = stub(console, 'log');
+      self.FIREBASE_APPCHECK_DEBUG_TOKEN = true;
+      initializeAppCheck(app, {
+        provider: new ReCaptchaV3Provider(FAKE_SITE_KEY)
+      });
+      // Ensure getDebugToken() call inside `initializeAppCheck()`
+      // has time to resolve, and double check its value matches that
+      // written to indexedDB.
+      expect(await getDebugToken()).to.equal(token);
+      expect(consoleStub.args[0][0]).to.include(token);
+      self.FIREBASE_APPCHECK_DEBUG_TOKEN = undefined;
+    });
+    it('warns about debug mode on second call', async () => {
```

Comment:
What does it test? 
I think we need a test to verify `initializeDebugMode` is called only once.

Reply:
Oh wow, the test caught the fact that I was not setting the `initialize` property... thanks.